lib/spack/spack/cmd/install.py

```diff
@@ -73,6 +77,11 @@ def setup_parser(subparser):
         '--fake', action='store_true', dest='fake',
         help="Fake install. Just remove prefix and create a fake file.")
 
+    subparser.add_argument(
+        '--install-status', '-I', action='store_true', dest='install_status',
+        help="Show spec before installing.")
```

Comment:
I agree, having two command which do the same thing is not good. I understand that `spack spec` may take some time for big packages, but IMO it's not a good enough reason to add this functionality to `spack install`.